testprogs/fuzz/fuzz_both.c

```diff
@@ -3,6 +3,7 @@
 #include <fcntl.h>
 #include <errno.h>
 #include <string.h>
+#include <unistd.h>
```

Comment:
With this change, you will need to arrange that the fuzz-testing infrastructure not be compiled on Windows, as Windows doesn't have unistd.h, or come up with an alternative for Windows.  Currently the Windows build is failing with this pull request.

Reply:
Yeah, I disabled fuzz targets build on Windows. Could you, please, restart netbsd-mips64 CI?